Code/GraphMol/Depictor/Templates.h

```diff
@@ -18,6 +18,7 @@
 #include <fstream>
 #include <unordered_map>
 
+namespace RDDepict {
 class CoordinateTemplates {
```

Comment:
This class needs to have `RDKIT_DEPICTOR_EXPORT` added. This is the reason for the win64_dll CI failure (sometimes it times out, sometimes it actually shows this error).